util/src/main/java/io/kubernetes/client/util/RetryUtils.java

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package io.kubernetes.client.util;
+
+import io.kubernetes.client.openapi.ApiException;
+import java.net.HttpURLConnection;
+
+public class RetryUtils {
+
+  public static <ApiType> ApiType retryUponConflict(
+      ApiInvocation<ApiType> apiTypeApiInvocation, int maxRetry) throws ApiException {
+    int retries = 0;
+    ApiException lastApiException;
+    while (true) {
+      try {
+        return apiTypeApiInvocation.call();
+      } catch (ApiException e) {
+        lastApiException = e;
+        if (HttpURLConnection.HTTP_CONFLICT == e.getCode()) {
+          retries++;
+          if (retries >= maxRetry) {
+            throw lastApiException;
```

Comment:
Since `lastException` always == `e` at this point in the code, just throw `e`

Reply:
done